stacks-common/src/address/c32.rs

```diff
@@ -180,7 +180,9 @@ const C32_CHARACTERS_MAP: [Option<u8>; 128] = [
 ];
 
 fn c32_encode(input_bytes: &[u8]) -> String {
-    let mut result = vec![];
+    // c32-encoded size is 160%  that of ASCII
```

Comment:
Saying 160% here is not obvious. I think it would be more clear to say something like:
```rust
    // ASCII characters are 8-bits and c32-encoding encodes 5-bits per
    // character, so the c32-encoded size should be (ascii size) * 8 / 5.
```